examples/demo13.py

```diff
@@ -21,8 +21,6 @@
 The thirteenth script in our tutorial about using Galsim in python scripts: examples/demo*.py.
 (This file is designed to be viewed in a window 100 characters wide.)
 
-This script currently doesn't have an equivalent demo*.yaml or demo*.json file.
-
 This script introduces non-idealities arising from NIR detectors, in particular those that will be
```

Comment:
I know this isn't something new, but reading this, I think it's missing a single high-level statement that the goal of the demo is to illustrate drawing a sky scene with features expected for images from the Roman Space Telescope.   (Currently it kind of implies that the goal is only to illustrate detector non-idealities, but there is more than that - Roman PSF and WCS etc.)  I suggest adding a 1-sentence preamble along those lines to demo13.py and demo13.yaml.

Reply:
Done.  I added:

This script is intended to produce a relatively realistic scene of galaxies and stars as will be observed by the Roman Space Telescope, including the Roman PSF, WCS, and various NIR detector effects.